README.md

```diff
@@ -1,47 +1,71 @@
+[`scheduled_install_day`]: #scheduled_install_day
+[`scheduled_install_hour`]: #scheduled_install_hour
+[`server_url`]: #server_url
 
-~~~
-$ puppet module install [--modulepath <path>] puppetlabs/wsus_client
-~~~
+[`target_group`]: #target_group
+
+#### Table of Contents
+
+1. [Module Description - What is the wsus_client module, and what does it do?](#module-description)
+  * [What wsus_client affects](#what-wsus_client-affects)
+2. [Setup - The basics of getting started with wsus_client](#setup)
+  * [Beginning with wsus_client](#beginning-with-wsus_client)
+3. [Usage - Configuration options and additional functionality](#usage)
+  * [Schedule automatic updates](#schedule-automatic-updates)
+4. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+5. [Limitations](#limitations)
+6. [Developments](#developments)
+
+## Module Description
 
-The above command also includes the optional argument to specify your Puppet master's `modulepath` as the location to install the module.
+The Windows Server Update Service (WSUS) lets Windows administrators manage operating system updates using their own servers instead of Microsoft's Windows Update servers.
 
+This module configures Puppet agents to schedule update downloads and installations from a WSUS server, manage user access to update settings, and configure automatic updates.
 
 ### What wsus_client affects
 
-The module updates registry keys located in `HKLM\Software\Policies\Microsoft\Windows\WindowsUpdate`
+This module modifies registry keys in `HKLM\Software\Policies\Microsoft\Windows\WindowsUpdate`. For details about how registry key-driven WSUS configuration works, see the [Microsoft TechNet documentation][TechNet].
+
+**Note**: Because this module modifies registry keys on clients, it is incompatible with Group Policy Objects that manage the same WSUS settings. **Do not use wsus_client to configure WSUS access or automatic updates if you use Group Policies to configure such options on clients**, as doing so can lead to unexpected behavior. Instead, consult Microsoft's documentation on [configuring automatic updates using Group Policy](https://technet.microsoft.com/en-us/library/dd939933.aspx).
+
+## Setup
+
+To use wsus_client, you must have a configured and running WSUS server, and your clients must run Windows Server 2003 or newer. For more information about deploying WSUS, see Microsoft's [WSUS Deployment Guide](https://technet.microsoft.com/en-us/library/dd939906.aspx).
+
+To install this module on your Puppet master, run this command:
+
+~~~
+$ puppet module install [--modulepath <PATH>] puppetlabs/wsus_client
+~~~
+
+If necessary, use the optional `--modulepath` argument to specify your Puppet master's `modulepath`.
 
 ### Beginning with wsus_client
 
-#### Set the WSUS server url to connect to
+To use wsus_client with its default settings, declare the `wsus_client` class with the WSUS server's url in the [`server_url`][] parameter.
```

Comment:
Disabling behaviors due to undefined (most enabled parameters) or explicitly disabled (`purge_values`) default parameter values is still setting default behaviors. These defaults, including the Windows defaults for disabled enabled parameters, should all be noted in the parameter references.

Should I make that more explicit here? Otherwise, I'd lean toward removing the bare `wsus_client` class example if it serves no functional purpose.